specs/latest/2.0/index.html

```diff
@@ -1274,6 +1274,9 @@ <h2 class="no-toc">Table of contents</h2>
         framebuffer. If the value of SAMPLE_BUFFERS for the read framebuffer is one and the value of
         SAMPLE_BUFFERS for the draw framebuffer is zero, the samples corresponding to each pixel location in
         the source are converted to a single sample before being written to the destination.
+
+        For any source pixel lying outside the read framebuffer, the corresponding destination pixel remains
+        untouched.
```

Comment:
I agree with @zhenyao. Let's discuss this issue at https://github.com/KhronosGroup/WebGL/pull/2098.